llvm/tools/llvm-objdump/COFFDump.cpp

```diff
@@ -301,11 +301,30 @@ static void printUnwindCode(ArrayRef<UnwindCode> UCs) {
     outs() << " " << (UCs[0].getOpInfo() ? "w/o" : "w")
            << " error code";
     break;
+
+  case UOP_Epilog:
+    if (SeenFirstEpilog) {
+      uint32_t Offset = (UCs[0].getOpInfo() << 8) |
+                        static_cast<uint32_t>(UCs[0].u.CodeOffset);
```

Comment:
Would it make sense to package up this math into a helper in Win64EH.h?

Reply:
Good idea.